Patch-Source: https://gitlab.com/kop316/vvmd/-/commit/235f096b.patch
Patch-Source: https://gitlab.com/kop316/vvmd/-/commit/d34732ce.patch
--
From 235f096b757ee7a9b8a0b06a1b7d1b7728a7822f Mon Sep 17 00:00:00 2001
From: Chris Talbot <chris@talbothome.com>
Date: Fri, 16 Feb 2024 14:21:51 -0700
Subject: [PATCH] service: remove debug lines

This causes it to not compile in i386
---
 src/service.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/service.c b/src/service.c
index a2a02e0..2dbdbe9 100644
--- a/src/service.c
+++ b/src/service.c
@@ -792,10 +792,8 @@ curl_string_cb (void  *data,
 {
   size_t realsize = size * nmemb;
   struct struct_string *mem = (struct struct_string *)userdata;
-  DBG ("received %lu", realsize);
 
   mem->response = g_string_append_len (mem->response, (char *)data, realsize);
-  //DBG("String so far: %s", mem->response->str);
   return realsize;
 }
 
-- 
GitLab

From d34732ce7e9f2089a4b5f50d518700f145816486 Mon Sep 17 00:00:00 2001
From: Chris Talbot <chris@talbothome.com>
Date: Fri, 16 Feb 2024 14:31:20 -0700
Subject: [PATCH] vvmutil: remove debug line

This doesnt allow compilation on i386
---
 src/vvmutil.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/vvmutil.c b/src/vvmutil.c
index 1a7dfa1..bf94f13 100644
--- a/src/vvmutil.c
+++ b/src/vvmutil.c
@@ -859,8 +859,6 @@ vvm_util_decode_vvm_single_email_attachment (const char *attachment,
           else if (found_attachment == FALSE)
             {
               attachment_line = i;
-              DBG ("Attachment contents on line: %d, length %lu",
-                   attachment_line, strlen (lines[i]));
               found_attachment = TRUE;
             }
         }
-- 
GitLab

